taxonomies/categories.txt

```diff
@@ -62775,6 +62804,7 @@ ciqual_food_code:en:11056
 ciqual_food_name:en:Mix of 4 spices
 ciqual_food_name:fr:Quatre épices
 
+<en:Condiments
```

Comment:
We can do that, and use a broad definition of condiment to include tomato sauces etc. Maybe something to let people know of in #taxonomies.